scripts/pull_and_format_datasets.py

```diff
@@ -0,0 +1,228 @@
+"""Pulls and formats data into alpaca format.
```

Comment:
Might want to specify it pulls specific datasets since this script isn't generic for any dataset

Reply:
Sure sounds good!